chatbot/src/main/java/chatbot/core/handlers/extractAutomaticallyWithKeywords/ExtractAutomaticallyWithKeywordsHandler.java

```diff
@@ -0,0 +1,45 @@
+package chatbot.core.handlers.extractAutomaticallyWithKeywords;
+
+import java.io.IOException;
+
+import org.apache.log4j.Logger;
+
+import chatbot.core.handlers.Handler;
+import chatbot.io.incomingrequest.IncomingRequest;
+import chatbot.io.response.ResponseList;
+
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.client.RestTemplate;
+
+@RestController
+public class ExtractAutomaticallyWithKeywordsHandler extends Handler{
+	private static Logger log = Logger.getLogger(ExtractAutomaticallyWithKeywordsHandler.class.getName());
+
+	private final RestTemplate restTemplate;
+
+	public ExtractAutomaticallyWithKeywordsHandler (RestTemplate restTemplate) {
+		this.restTemplate = restTemplate;
+}
+
+	public ResponseList search(IncomingRequest request) throws IOException {
```

Comment:
is there a reason you do it like this?
Why you didnt do it in the default way, like in the spring (https://spring.io/guides/gs/rest-service/)?